tensorflow_datasets/core/visualization/visualizer.py

```diff
@@ -21,15 +21,47 @@
 from __future__ import division
 from __future__ import print_function
 
+import os
+import inspect
+
 import abc
 from typing import Any
 
 import six
 import tensorflow.compat.v2 as tf
 
+import tensorflow_datasets as tfds
 from tensorflow_datasets.core import dataset_info
 
 
+def extract_dataset_type(full_name):
```

Comment:
Let's remove this as this isn't used anywhere

Reply:
Done.